src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -611,7 +617,7 @@ private void OnResetMarkNodesAsDirty(bool markNodesAsDirty)
                 workspace.ResetEngine(EngineController, markNodesAsDirty);
         }
 
-        public void SetRunEnabledBasedOnContext(View newView)
+        public void SetRunEnabledBasedOnContext(View newView, bool doContextAvailable)
```

Comment:
Add new flag that allow us to show "Dynamo available" when going from bad Revit context to good Revit context.